### PR TITLE
Fix csv extension check

### PR DESCRIPTION
**Pull request by skmatz**
## What does this PR do?

Fix csv extension check due to a common mistake in Python tuple literal.

```python
if extension.lower() in ('csv'):
	...
```

## Before submitting

- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Review by Borda — APPROVED** [review]

**Review by ananthsub — APPROVED** [review]
is there a test for this?

**Comment by skmatz** [comment]
OK, I will create CHANGELOG soon.

File: pytorch_lightning/core/saving.py [hunk 1]
```diff
@@ -136,7 +136,7 @@ def load_from_checkpoint(
 
         if hparams_file is not None:
             extension = hparams_file.split('.')[-1]
-            if extension.lower() in ('csv'):
+            if extension.lower() in ('csv',):
```
Comment (rohitgr7) [inline pytorch_lightning/core/saving.py:139]:
```suggestion
            if extension.lower() == 'csv':
```

**Review by tchaton — APPROVED** [review]
LGTM !

**Comment by skmatz** [comment]
It seems that there is no test originally, do I need to write a new test?

**Review by carmocca — APPROVED** [review]

**Pull request merged**